runBiDiServer.sh

```diff
@@ -19,7 +19,13 @@ if [[ $# -gt 0 && ("$1" == "-h" || "$1" == "--help") ]]; then
 fi
 
 # The chrome release channel to use: `stable`, `beta`, `canary`, `dev`.
-readonly CHANNEL="${CHANNEL:-dev}"
+CHANNEL="${CHANNEL:-local}"
+
+# Run the pinned CfT Cannary by default
+if [ $CHANNEL == "local" ]; then
+  export BROWSER_BIN=$(node install-browser.mjs ~/.cache/chromium-bidi --shell)
+  CHANNEL='canary'
```

Comment:
```suggestion
  CHANNEL='dev'
```

Since canary does not exist on Linux, please keep defaulting to `dev` here.

Reply:
This change is correct we pass it down into some parser that expects the only `stable`, `dev` etc.
So `canary` is the closes correct one.

Reply:
OK, I see what you mean. Sorry, disregard.

Ideally I would eliminate `CHANNEL` but let's keep this out of scope of this PR.